src/test/java/millom/sandbox/mapper/NasaMapperTest.java

```diff
@@ -0,0 +1,247 @@
+      + "            \"sol\": \"3521\",\n"
+      + "            \"ls\": \"258\",\n"
+      + "            \"season\": \"Month 9\",\n"
+      + "            \"min_temp\": \"-69\",\n"
+      + "            \"max_temp\": \"-14\",\n"
+      + "            \"pressure\": \"882\",\n"
+      + "            \"pressure_string\": \"Higher\",\n"
+      + "            \"abs_humidity\": \"--\",\n"
+      + "            \"wind_speed\": \"--\",\n"
+      + "            \"wind_direction\": \"--\",\n"
+      + "            \"atmo_opacity\": \"Sunny\",\n"
+      + "            \"sunrise\": \"05:51\",\n"
+      + "            \"sunset\": \"18:08\",\n"
+      + "            \"local_uv_irradiance_index\": \"High\",\n"
+      + "            \"min_gts_temp\": \"-77\",\n"
+      + "            \"max_gts_temp\": \"8\"\n"
+      + "        }\n"
+      + "]\n"
+      + "}";
+
+  public static final String RESPONSE_MISSING_JSON_PROPERTY="{\n"
+      + "    \"soles\": [\n"
+      + "        {\n"
+      + "            \"id\": \"3345\",\n"
+      + "            \"terrestrial_date\": \"2022-07-03\",\n"
+      + "            \"sol\": \"3522\",\n"
+      + "            \"ls\": \"258\",\n"
+      + "            \"season\": \"Month 9\",\n"
+      + "            \"min_temp\": \"-67\",\n"
+      + "            \"max_temp\": \"-10\",\n"
+      + "            \"pressure\": \"870\",\n"
+      + "            \"pressure_string\": \"Higher\",\n"
+      + "            \"abs_humidity\": \"--\",\n"
+      + "            \"wind_speed\": \"--\",\n"
+      + "            \"wind_direction\": \"--\",\n"
+      + "            \"atmo_opacity\": \"Sunny\",\n"
+      + "            \"sunrise\": \"05:52\",\n"
+      + "            \"sunset\": \"18:09\",\n"
+      + "            \"local_uv_irradiance_index\": \"High\",\n"
+      + "            \"min_gts_temp\": \"-72\",\n"
+      + "            \"max_gts_temp\": \"8\"\n"
+      + "        },\n"
+      + "        {\n"
+      + "            \"id\": \"3344\",\n"
+      + "            \"terrestrial_date\": \"2022-07-02\",\n"
+      + "            \"sol\": \"3521\",\n"
+      + "            \"ls\": \"258\",\n"
+      + "            \"season\": \"Month 9\",\n"
+      + "            \"min_temp\": \"-69\",\n"
+      + "            \"max_temp\": \"-14\",\n"
+      + "            \"pressure\": \"882\",\n"
+      + "            \"pressure_string\": \"Higher\",\n"
+      + "            \"abs_humidity\": \"--\",\n"
+      + "            \"wind_speed\": \"--\",\n"
+      + "            \"wind_direction\": \"--\",\n"
+      + "            \"atmo_opacity\": \"Sunny\",\n"
+      + "            \"sunrise\": \"05:51\",\n"
+      + "            \"sunset\": \"18:08\",\n"
+      + "            \"local_uv_irradiance_index\": \"High\",\n"
+      + "            \"min_gts_temp\": \"-77\",\n"
+      + "            \"max_gts_temp\": \"8\"\n"
+      + "        }\n"
+      + "]\n"
+      + "}";
+
+  // test json string mapping correct to the weather object
```

Comment:
Are these comments above the test methods needed? What about giving the test methods a self explanatory name?

Reply:
Changed to giving the test methods a self explanatory name.

 ```
@DisplayName("TEST JSON STRING MAPPING CORRECT TO THE WEATHER OBJECT")
void deserializeWeather() throws InvalidWeatherJsonException {

}
```